### PR TITLE
QOL addtions to the readme

### PR DESCRIPTION
**Pull request by jpburnett**
- Added some instructions to help those unfamiliar with make/build process, and thought it would help. 
- Added fancy badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +22,87 @@ source code base for general changes.
...
+
+Create your own fork of the repository at `https://github.com/zeldaret/mm`. Then clone your fork where you wish to have the project, with the command:
+
+```bash
+git clone https://github.com/<YOUR_USERNAME>/mm.git
+```
+
+#### 3. Prepare a base ROM
+
+Copy your ROM to inside the root of this new project directory, and rename the file to "baserom.z64".
```
Comment (Kenix3) [inline README.md:74]: This should probably accompany a rename of the baserom to reflect the version. The naming format we will be using in the future can be found below. We also want to make sure the Jenkinsfile is updated with the new rom destination as well.
https://github.com/zeldaret/mm/blob/e97f10a6fa573e8b817ed03a4674d52b1818dafa/Jenkinsfile#L10

**Review by Kenix3 — CHANGES_REQUESTED** [review]

**Comment by jpburnett** [comment]
also added the shield icon that was made in the progress update should resolve #100

File: README.md (hunk 1 above)
Comment (jpburnett) [inline README.md:77]:
```suggestion
Copy your ROM to inside the root of this new project directory, and rename the file of the baserom to reflect the version of ROM you are using. ex: `mm.us.rev1.z64`.
```

Comment (jpburnett) [reply README.md:77]:
```suggestion
Copy your ROM to inside the root of this new project directory, and rename the file of the baserom to reflect the version of ROM you are using. ex: `mm.us.rev1.z64`
```
So a change to something like this?

Comment (Kenix3) [reply README.md:77]: Yes.

Comment (Kenix3) [reply README.md:77]: I forgot one more line that needs to change:
https://github.com/zeldaret/mm/blob/e97f10a6fa573e8b817ed03a4674d52b1818dafa/Makefile#L69

**Comment by Kenix3** [comment]
Couple issues left.

This needs to be updated to the new rom name as well.
https://github.com/zeldaret/mm/blob/master/first_diff.py#L37


We have a build error. I'm looking into where it gets that file name from.

```
2a0a8acb61538235bc1094d297fb6556  mm.us.rev1.rom.z64
md5sum: rom.z64: No such file or directory
rom.z64: FAILED open or read
md5sum: WARNING: 1 listed file could not be read
make[1]: *** [Makefile:141: mm.us.rev1.rom.z64] Error 1
make[1]: *** Waiting for unfinished jobs....
7c6bfd6daf33ebfc0145a97c611cb3d8  mm.us.rev1.rom_uncompressed.z64
md5sum: rom_uncompressed.z64: No such file or directory
rom_uncompressed.z64: FAILED open or read
md5sum: WARNING: 1 listed file could not be read
make[1]: *** [Makefile:134: mm.us.rev1.rom_uncompressed.z64] Error 1
make[1]: Leaving directory '/home/jenkins/workspace/MM_PR-104'
make: *** [Makefile:198: init] Error 2
```

**Comment by hensldm** [comment]
The builds look to be failing since the .md5 checksums have not been updated to use the new outputed rom name.
`2a0a8acb61538235bc1094d297fb6556  rom.z64` would have to change to `2a0a8acb61538235bc1094d297fb6556  mm.us.rev1.rom.z64`

**Review by Kenix3 — APPROVED** [review]
Looks good to me!

**Pull request merged**